opencti-platform/opencti-front/src/utils/tests/useOverviewLayoutCustomization.test.ts

```diff
@@ -0,0 +1,23 @@
+import { describe, it, expect } from 'vitest';
+import useOverviewLayoutCustomization, { OverviewLayoutCustomizationSettingsConfigurationParameters } from '../hooks/useOverviewLayoutCustomization';
+
+describe('useOverviewLayoutCustomization', () => {
+  it('should provide overview layout customization settings by block for a given entity type', () => {
+    const threatActorIndividualEntityType = 'Threat-Actor-Individual';
+    const expectedThreatActorIndividualConfiguration = new Map<string, OverviewLayoutCustomizationSettingsConfigurationParameters>();
+    expectedThreatActorIndividualConfiguration.set('details', { order: 1, width: 6 });
+    expectedThreatActorIndividualConfiguration.set('basicInformation', { order: 2, width: 6 });
+    expectedThreatActorIndividualConfiguration.set('demographics-biographics', { order: 3, width: 6 });
+    expectedThreatActorIndividualConfiguration.set('latestCreatedRelationships', { order: 5, width: 6 });
+    expectedThreatActorIndividualConfiguration.set('latestContainers', { order: 6, width: 6 });
+    expectedThreatActorIndividualConfiguration.set('externalReferences', { order: 7, width: 6 });
+    expectedThreatActorIndividualConfiguration.set('mostRecentHistory', { order: 8, width: 6 });
+    expectedThreatActorIndividualConfiguration.set('notes', { order: 9, width: 12 });
+
+    expect(
+      useOverviewLayoutCustomization(threatActorIndividualEntityType),
+    ).toEqual(
+      expectedThreatActorIndividualConfiguration,
```

Comment:
I don't see how this test will work in the future... I think you should test that keys exists, that width are set or stuff like that as it will be updatable